testing/ef_tests/src/cases/rewards.rs

```diff
@@ -108,53 +104,30 @@ impl<E: EthSpec> Case for RewardsTest<E> {
 
     fn result(&self, _case_index: usize, fork_name: ForkName) -> Result<(), Error> {
         let mut state = self.pre.clone();
+
+        // NOTE: We cannot run these tests for forks other than phase0 because single-pass epoch
+        // processing cannot expose these individual deltas. There is no point maintaining a
+        // separate implementation of rewards processing that will not be used in prod.
+        if fork_name != ForkName::Base {
+            return Ok(());
+        }
```

Comment:
For post-base forks, assert the sum of all deltas instead of skipping

Reply:
Great idea, implemented in the latest commit. Just had to skip 1 test due to https://github.com/ethereum/consensus-specs/issues/3593.